### PR TITLE
OP-187: IMIS web and policies app: Renewal extracts dont work

### PR DESCRIPTION
**Pull request by malinowskikam**
Changes:
- Updated renewal extreact import feature to work with Android 10/11 filesystem security
- Chaged hard-coded strings used in renewal extreact import to string resources
- Fixed missing string resources in `Settings` page

Maintenance:
- removad all references to `Jcenter` repository (read-only from March 31st, 2021, replaced with `Maven`)
- Bumped Java version to 11 (LTS) (Packaged with newest stable version of Android Studio, also required by one dependency, does not break compability with older versions of Android)
- Bumped Gradle varsion to 7.0.2

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -15,12 +15,8 @@
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.MANAGE_EXTERNAL_STORAGE" />
-    <uses-permission
-        android:name="android.permission.READ_CALL_LOG"
-        tools:node="remove" />
-    <uses-permission
-        android:name="android.permission.READ_CONTACTS"
-        tools:node="remove" />
+    <uses-permission android:name="android.permission.READ_CALL_LOG"/>
```
Comment (dragos-dobre) [inline app/src/main/AndroidManifest.xml:18]: why we have this permission again?

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (malinowskikam) [reply app/src/main/AndroidManifest.xml:18]: I'm not sure, but from what I checked it seems that access to the phone app is needed to read the IMEI of the phone, and it was a feature. I don't think we need it currently. Should I remove it?

**Comment by dragos-dobre** [comment]
@malinowskikam Can you please check the conflicts? Thanks.

**Comment by malinowskikam** [comment]
@dragos-dobre I fixed the conflicts

**Review by dragos-dobre — APPROVED** [review]

**Pull request merged**